src/DotNest.Core.SDK.Web/Program.cs

```diff
@@ -0,0 +1,31 @@
+using Lombiq.Hosting.Tenants.Management.Extensions;
+using NLog.Web;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Logging.ClearProviders();
+builder.Host.UseNLog();
+
+var configuration = builder.Configuration;
+
+builder.Services.AddOrchardCms(orchardCoreBuilder =>
+{
+    orchardCoreBuilder
+        .AuthorizeApiRequestsIfEnabled(builder.Configuration)
+        .AllowMiniProfilerOnAdmin()
+        .AddDatabaseShellsConfigurationIfAvailable(configuration)
+        .ConfigureSmtpSettings(overrideAdminSettings: false)
+        .ConfigureUITesting(configuration, enableShortcutsDuringUITesting: true);
+
+    if (!configuration.IsUITesting())
```

Comment:
We don't have any UI testing here and won't be, so please delete the if statement